WordPress/Classes/Utility/Logging/SentryStartupEvent.swift

```diff
@@ -26,11 +34,11 @@ startup time. This will block the thread. Do not use unless you're sure.
         }
         let semaphore = DispatchSemaphore(value: 0)
         let event = Event(level: .debug)
-        let lastError = errors.removeLast()
-        event.message = "\(title): \(lastError)"
+        event.message = title
         for error in errors {
             let breadcrumb = Breadcrumb(level: .debug, category: "Startup")
-            breadcrumb.message = error
+            breadcrumb.message = error.0
+            breadcrumb.data = error.1
```

Comment:
You can make this into named variables as you iterate for a bit more clarity:

![Screen Shot 2020-02-11 at 09 53 26](https://user-images.githubusercontent.com/8739/74222250-5c06df80-4cb4-11ea-9ec0-84bae4c872d5.png)

Reply:
Ah nice, thank you 👍